src/express/utils/get-request-headers.ts

```diff
@@ -8,12 +8,12 @@ export function getRequestHeaders(
   const multiValueHeaders: Record<string, string[]> = {};
 
   for (const [key, value] of Object.entries(req.headers)) {
-    if (Array.isArray(value)) {
-      multiValueHeaders[key] = value;
-    } else if (typeof value === `string`) {
-      headers[key] = value;
+    if (value) {
+      const values = [value].flat();
+      multiValueHeaders[key] = values;
+      headers[key] = values[values.length - 1] || ``;
     }
-  }
+  };
```

Comment:
I think this semi will result in a Prettier format error.

Reply:
gone